app/controllers/prospects_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ProspectsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
That's ok. My goal is that the closed version coexists with the open version in a single code base.